src/Symfony/Component/Debug/Exception/FlattenException.php

```diff
@@ -33,7 +33,7 @@ class FlattenException
     private $file;
     private $line;
 
-    public static function create(\Exception $exception, $statusCode = null, array $headers = array())
+    public static function create(\Throwable $exception, $statusCode = null, array $headers = array())
```

Comment:
This change will break applications that extend this class and override the `create` method.